fabric-transitive-access-wideners-v1/build.gradle

```diff
@@ -75,6 +77,32 @@ def generateBlockConstructors(List<String> lines, FileSystem fs) {
 		}
 }
 
+def generateItemConstructors(List<String> lines, FileSystem fs) {
+	lines.add("# Constructors of non-abstract item classes")
+	Files.list(fs.getPath("net/minecraft/item"))
```

Comment:
This is fine for now, in the future it may be better to check the class hierarchy similar to how I did it in the data gen module.